api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -90,9 +95,17 @@ def publish_cdr(cmd_name, args)
     # If you receive a prompt from "bq" for selecting a default project ID, just
     # hit enter.
     # TODO: Figure out how to prepopulate this value or disable interactivity.
-    common.run_inline %W{bq mk -f --dataset #{dest_dataset}}
+
+    # Copy through an intermediate project and delete after, per
+    # https://docs.google.com/document/d/1EHw5nisXspJjA9yeZput3W4-vSIcuLBU5dPizTnk1i0/edit
+    common.run_inline %W{bq mk -f --dataset #{ingest_dataset}}
     common.run_inline %W{./copy-bq-dataset.sh #{source_dataset} #{dest_dataset} #{env.fetch(:source_cdr_project)}}
```

Comment:
I may be confused, but should this be like so?
```suggestion
    common.run_inline %W{./copy-bq-dataset.sh #{source_dataset} #{ingest_dataset} #{env.fetch(:source_cdr_project)}}
```